docs/tutorial/comparisons.md

```diff
@@ -0,0 +1,621 @@
+# Comparisons
+
+Comparisons are complicated. They are not native to TFHE, so they need to be implemented using existing native operations (i.e., additions, clear multiplications, negations, table lookups). Concrete offers three different implementations for performing comparisons.
```

Comment:
Comparisons are not native operations in Concrete and are implemented ...

Reply:
Sure.